pom.xml

```diff
@@ -12,12 +12,14 @@
         <maven.compiler.source>21</maven.compiler.source>
         <maven.compiler.target>21</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <tinkar-core.version>1.67.0</tinkar-core.version>
+        <tinkar-core.version>1.71.0-SNAPSHOT</tinkar-core.version>
```

Comment:
Can you change this to 1.70.0? It's best to not rely on snapshot builds when checking in.